src/index.js

```diff
@@ -3,11 +3,25 @@ import { AppContext, AppProvider, withApp } from "./AppProvider";
 import Stage from "./Stage";
 import { TYPES } from "./types";
 import { render } from "./render";
+import { usePixiApp, usePixiTicker, usePreviousProps, usePixiAppCreator } from "./hooks";
 import { applyProps, unstable_batchedUpdates } from "./ReactPixiFiber";
 
 /* Public API */
 
-export { AppContext, AppProvider, CustomPIXIComponent, Stage, applyProps, render, withApp, unstable_batchedUpdates };
+export {
+  AppContext,
+  AppProvider,
+  CustomPIXIComponent,
+  Stage,
+  applyProps,
+  render,
+  withApp,
+  unstable_batchedUpdates,
+  usePixiApp,
+  usePixiTicker,
+  usePreviousProps,
```

Comment:
This is mostly for internal use, I think we should not export it. Even though it's simple and useful it is not related to the API that is already surfaced to users.